figlet/__init__.py

```diff
@@ -62,4 +69,23 @@ def check_font_rendering(font, text):
         output = ""
         for line in lines:
             output += line
-        check50.run(f"python3 figlet.py -f {font}").stdin(text, prompt=False).stdout(regex(output), output, regex=True).exit(0)
+    
+    if font == "random":
+        run = check50.run("python3 testing.py")
+    else:
+        run = check50.run(f"python3 figlet.py -f {font}")
+        
+    try:
+        exit = run.exit(timeout=1)
```

Comment:
I'm not too sure about the `timeout` length, as `testing.py` scans the `figlet.py` file.  
1 second seems enough, but I don't know in what kind of environment check50 is being run.  
Maybe executing thousands of tests at once could have some impact on this 🤔.